src/components/navbar/Layout.js

```diff
@@ -1,33 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Drawer from 'material-ui/Drawer';
-import List from 'material-ui/List';
+import List, { ListItem, ListItemText } from 'material-ui/List';
 import Divider from 'material-ui/Divider';
+import {navigateTo} from '../../utils/helpers'
+import Paper from 'material-ui/Paper'
 
 export default class Layout extends React.Component {
   render() {
     return (
-      <Drawer variant="permanent" anchor="left">
-        <div />
+      <Paper style={styles.sideBar}>
         <Divider />
-        <List>{'hello'}</List>
-        <List>{'hello'}</List>
-        <List>{'hello'}</List>
+        <List>
+          <ListItem button onClick={navigateTo.bind(this, "/", "")}>
```

Comment:
instead of `navigateTo.bind` use arrow function instead. So onClick={() => navigateTo("/")}

Reply:
Ok I changed the functions